code/game/area/areas.dm

```diff
@@ -322,6 +323,19 @@ var/area/space_area
 		updateicon()
 	return
 
+/area/proc/get_ambience_list()
+	//Check if the area has an AI and add the appropriate ambience
+	var/list/ambience_list = list()
+	ambience_list.Add(ambient_sounds)
+	for(var/mob/living/silicon/ai/AI in player_list)
+		if(get_area(AI) == src)
+			ambience_list.Add(/datum/ambience/AI, /datum/ambience/AI/safe, /datum/ambience/AI/back)
+			if(AI?.laws.name == "Asimov's Three Laws of Robotics")
```

Comment:
malf will still trigger this, also metaway to know they're on asimov

Reply:
The AI ambience no longer fires on malf, and the asimov one was made much rarer as a compromise.